lib/entities/quote/DutchQuote.ts

```diff
@@ -358,15 +358,19 @@ export abstract class DutchQuote<T extends DutchQuoteRequest> implements IQuote
   }
 
   // Returns the number of gas units extra required to execute this quote through UniswapX
-  static getGasAdjustment(classicQuote: ClassicQuote): BigNumber {
+  static getGasAdjustment(classicQuote: ClassicQuote, adjustmentBps?: number): BigNumber {
     let result = BigNumber.from(0);
 
     // fill contract must unwrap WETH output tokens
     if (classicQuote.request.info.tokenOut === NATIVE_ADDRESS) {
       result = result.add(WETH_UNWRAP_GAS);
     }
 
-    return result.add(UNISWAPX_BASE_GAS);
+    result.add(UNISWAPX_BASE_GAS);
```

Comment:
```suggestion
    result = result.add(UNISWAPX_BASE_GAS);
```

Reply:
Same comment as above! you got auto pinged for review even though this is still a draft